crates/driver/src/infra/solver/dto/auction.rs

```diff
@@ -15,7 +15,14 @@ use {
             serialize,
         },
     },
+    app_data::AppDataHash,
     indexmap::IndexMap,
+    model::{
+        interaction::InteractionData,
+        order::{BuyTokenDestination, SellTokenSource},
+        signature::Signature,
```

Comment:
What happened to not using the `model` crate here? Were there issues with that after all?

Reply:
No issues, just I thought it was needed to be removed only from `solvers-dto` create. Removing it now, no probs.